packages/wonder-blocks-button/docs.md

```diff
@@ -1,11 +1,6 @@
 **WARNING: Button is not finished, do not use!**
 
-Default Button:
-```js
-<Button>default button</Button>
-```
+A `Button` is a clickable button element consisting of a `ButtonBehavior` (TODO(yejia)) surrounding a `ButtonCore`. `ButtonBehavior` handles interactions and state changes. `ButtonBehavior` is a stateless component which displays the different states the `Button` can take.
 
-Caution Button:
-```js
```

Comment:
Also, can you add some samples of the small sizes to button-core.md? I'm curious to see what they look like (especially for the focus/hover state).